src/Collection/Collection.router.ts

```diff
@@ -49,7 +49,7 @@ export class CollectionRouter extends Router {
     this.router.put(
       '/collections/:id',
       withAuthentication,
-      server.handleRequest(this.upsertCollection)
+      server.handleRequest(this.upsertCollection.bind(this))
```

Comment:
The bind is not necessary, we're using the already bound method syntax `= () =>`

Reply:
Hmmm, we are not using it that way. 

I had to do it because I was getting:
<img width="544" alt="Screen Shot 2021-03-01 at 21 37 17" src="https://user-images.githubusercontent.com/7549152/109578713-7174c480-7ad6-11eb-8efa-95c62e72c1fd.png">